site/en/blog/sqlite-wasm-in-the-browser-backed-by-the-origin-private-file-system/index.md

```diff
@@ -0,0 +1,260 @@
+---
+layout: 'layouts/blog-post.njk'
+title: SQLite Wasm in the browser backed by the Origin Private File System
+subhead: >
+  Use SQLite to handle all your storage needs performantly on the web.
+date: 2023-01-11
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/l5kRHOrUI9mQmwOTJKr7.jpg
+alt: Library symbolizing a database.
+authors:
+  - thomassteiner
+tags:
+  - capabilities
+---
+
+{% Aside 'success' %} In our blog post
+[Deprecating and removing Web SQL](/blog/deprecating-web-sql/), we promised a replacement for Web
+SQL based on SQLite. The SQLite Wasm library with the Origin Private File System persistence backend
+is our fulfillment of this promise. {% endAside %}
+
+## About SQLite
+
+[SQLite](https://sqlite.org/) is a popular, [open-source](https://sqlite.org/src/tree), lightweight,
+embedded relational database management system. Many developers use it to store data in a
+structured, easy-to-use manner. Because of its small size and low memory requirements, SQLite is
+often leveraged as a database engine in mobile devices, desktop applications, and web browsers.
+
+One of the key features of SQLite is that it is a serverless database, which means that it does not
+require a separate server process to operate. Instead, the database is stored in a single file on
+the user's device, making it easy to integrate into applications.
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/eDfwxFJirAZ6hqdt6Z8Y.png", alt="SQLite logo.", width="220", height="101" %}
+
+## SQLite based on Web Assembly
+
+There are a number of unofficial SQLite versions based on Web Assembly (Wasm), allowing it to be
+used in web browsers, for example, [sql.js](https://github.com/sql-js/sql.js). The
+[sqlite3 WASM/JS subproject](https://sqlite.org/wasm/doc/tip/about.md) is the first effort that is
+"officially" associated with the [SQLite project](https://sqlite.org/) makingWasm builds of the
```

Comment:
```suggestion
"officially" associated with the [SQLite project](https://sqlite.org/) making Wasm builds of the
```

Reply:
also why the scare quotes round officially?

Reply:
Quotes be gone.